lib/src/layer_properties.dart

```diff
@@ -1751,6 +1751,161 @@ class FillLayerProperties implements LayerProperties {
   }
 }
 
+class FillExtrusionLayerProperties implements LayerProperties {
```

Comment:
no need to write this by hand: 
add `fill-extrusion` to `scripts/lib/generate.dart` and run it to generate this code